pages/Blistering-Ingot.md

```diff
@@ -0,0 +1,7 @@
+Blistering Ingots are [Resource](https://github.com/Slimefun/Slimefun4/wiki/Resources)s that can be used to craft [reactor-related](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) items.
```

Comment:
```suggestion
Blistering Ingots are [Resources](https://github.com/Slimefun/Slimefun4/wiki/Resources) that can be used to craft [Reactor](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines#energy-generation) related items and [End-Game Machines](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines).
```

Reply:
> 
> 
> A suggestion maybe would be to put all the Carbon variants (Carbon, Carbon-Chunk, Compressed-Carbon) on one page, instead of spreading out over multiple pages.
Yeah, I can't really control that though. I think it has something to do with the wiki's setup? If it's determined that putting them together is better I could fix it.
> 
> Also, the Blistering Ingot has multiple tiers, we could perhaps mention those on the wiki too?
> (Doesnt have to be in this PR, just generally as a suggestion)
> 
Yeah, it does. I can't remember off the top of my head what the different amounts do though...
> Ive reviewed with suggestion blocks everywhere, you dont have to use my proposed changes, but it indicates how I would change it.
> And thank you for all the new pages!
No problem, thanks for the feedback!

Reply:
> Yeah, I can't really control that though. I think it has something to do with the wiki's setup? If it's determined that putting them together is better I could fix it.

You actually can, the wiki determines which pages are not created yet by checking which pages are all linked in other pages, so everywhere where carbon is mentioned youd have to update the link, I would just merge this PR and then Ill take care of putting it on one page if thats okay? Or you can do it yourself, whichever you prefer.